src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -278,6 +280,315 @@ struct mac_addr_list
   }
 };
 
+struct aim_counter
```

Comment:
You may want a comment about this struct.  I know this is not followed consistently, but comments are always good.

Reply:
I have added a comment